test/k8sT/manifests/hubble-cli.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: DaemonSet
```

Comment:
Maybe this is not the right place to ask this but once we have multi-node support shouldn't this be a Deployment instead?

Reply:
Most likely yes. Right now however, we still rely on per-node information and thus need a DaemonSet.